.github/workflows/daily_ci.yml

```diff
@@ -63,12 +64,21 @@ jobs:
     with:
       dafny: ${{needs.getVersion.outputs.version}}
 
-  daily-dafny-test-vectors:
+  daily-dafny-keyring-test-vectors:
     if: github.event_name != 'schedule' || github.repository_owner == 'aws'
     needs: getVersion
-    uses: ./.github/workflows/library_interop_test_vectors.yml
+    uses: ./.github/workflows/library_interop_keyring_test_vectors.yml
     with:
       dafny: ${{needs.getVersion.outputs.version}}
+    secrets: inherit
```

Comment:
we might not need this anymore... since the repos we are accessing are now public

Reply:
I will merge as-is and we can fix later.
CI takes over an hour, and this is blocking me from releasing new Python MPL/ESDK.
Plus, this is already committed in other places.